test/controllers/groups/samples_controller_test.rb

```diff
@@ -38,5 +38,60 @@ class SamplesControllerTest < ActionDispatch::IntegrationTest
            as: :turbo_stream
       assert_response :unprocessable_entity
     end
+
+    test 'should select samples' do
+      timestamp = DateTime.current
+      get select_group_samples_url(@group),
+          params: { select: true, timestamp: timestamp },
+          as: :turbo_stream
+      assert_response :success
+    end
+
+    test 'should not select samples without permission' do
+      sign_in users(:steve_doe)
+      timestamp = DateTime.current
+      get select_group_samples_url(@group),
+          params: { select: true, timestamp: timestamp },
+          as: :turbo_stream
+      assert_response :unauthorized
+    end
+
+    test 'should handle metadata template none' do
```

Comment:
Can you add these tests for the projects samples controller test as well

Reply:
Added in 76639e2

Reply:
Added in a25924ab157f41ae4d3af0596d7b3c68040e2caf